src/main/java/jmgomez/policyapi/service/PolicyService.java

```diff
@@ -23,25 +28,56 @@ public class PolicyService implements IPolicyService {
     @Autowired
     private AccidentMapper accidentMapper;
 
+    @Autowired
+    private CircuitBreakerFactory circuitBreakerFactory;
+
     @Override
+    @CircuitBreaker(name = "policiesCB", fallbackMethod = "fallBackGetPolicies")
```

Comment:
Do we really need fallback methods? per method? sending a 404 to the client? Are we into lying to our clients now? This is not ok. You aren't providing any configuration either. What's the circuit breaking strategy here? Also, I need a description in the PR showing you tried a circuit breaker and you know how it works.

Reply:
I'm trying to learn how it works but rightnow it's not working. Im working on it